models/classes/LtiLaunchData.php

```diff
@@ -143,63 +143,90 @@ public static function fromRequest(common_http_Request $request)
     }
 
     public static function fromLti1p3MessagePayload(
-        LtiMessagePayloadInterface $ltiMessagePayload,
+        LtiMessagePayloadInterface $payload,
```

Comment:
Why did you change that variable name? It can cause more harm then good.

Reply:
it is shorter and still meaningful cause has type hint